### ISSUE TITLE
1.64 etc+kaspa will be stuck in a few hours

### ISSUE THREAD

**Issue by wen936988** [Lolliedieb/lolMiner-releases#1818]
1.64 etc+kaspa will be stuck in a few hours，Must be closed and reopened.As shown in the figure
![3186e545dcf6138212a006f0e6809f5](https://user-images.githubusercontent.com/37339187/207191759-9ed9854b-7658-4b19-b825-1c6ce0cca98a.png)

**Comment by lvgucciprada** [comment]
Me too. Do you use F2Pool or Ethermine?

**Comment by dawidmosk** [comment]
Same issue here with pool ezil and woolypooly, on 1.63 was working ok

**Comment by lvgucciprada** [comment]
> 

If I do single mining on kas, there’s no problem. Dual mining on etc and kas, stuck randomly, no matter f2pool or ethermine.

**Comment by jgonzis** [comment]
> 

One of the gpu has crashed... seen Memory OC could it be someone of them quite high

**Comment by dawidmosk** [comment]
For example i did not change anything last time.

**Comment by lvgucciprada** [comment]
Yep, nothing changes except update to 1.64. But now I got the same problem after roll back to 1.63.

**Comment by jgonzis** [comment]
> Yep, nothing changes except update to 1.64. But now I got the same problem after roll back to 1.63.

But new lolMiner makes more stress in gpu, so it could need a little bit less core offset

**Comment by dawidmosk** [comment]
> Yep, nothing changes except update to 1.64. But now I got the same problem after roll back to 1.63.

Try 1.62 - for me working now for ~24h without problems..testing (changing from 1.64 to 1.63 had same problem)

**Comment by dawidmosk** [comment]
> > Yep, nothing changes except update to 1.64. But now I got the same problem after roll back to 1.63.
> 
> But new lolMiner makes more stress in gpu, so it could need a little bit less core offset

i will try too, core or mem or both?

**Comment by jgonzis** [comment]
> dawidmosk

Do a mix... a bit lower in both

**Comment by dawidmosk** [comment]
> > dawidmosk
> 
> Do a mix... a bit lower in both

ok, testing(less ~50) - for now 2 hours - checking next values ;-)

**Comment by lvgucciprada** [comment]
> 1.64 etc+kaspa will be stuck in a few hours，Must be closed and reopened.As shown in the figure
> ![3186e545dcf6138212a006f0e6809f5](https://user-images.githubusercontent.com/37339187/207191759-9ed9854b-7658-4b19-b825-1c6ce0cca98a.png)

Hey, bro, may I add you telegram friend?

**Comment by jgonzis** [comment]
> > 1.64 etc+kaspa will be stuck in a few hours，Must be closed and reopened.As shown in the figure
> > ![3186e545dcf6138212a006f0e6809f5](https://user-images.githubusercontent.com/37339187/207191759-9ed9854b-7658-4b19-b825-1c6ce0cca98a.png)
> 
> Hey, bro, may I add you telegram friend?

That normally is something about the core offset. If you don't have --watchdog exit when it crash the nvidia gpu doesn't stop the miner. You should check the log until.stop mining and it will tell you which GPU has crashed
